vowpalwabbit/vw.h

```diff
@@ -190,6 +191,13 @@ inline uint64_t chain_hash(vw& all, const std::string& name, const std::string&
       all.parse_mask;
 }
 
+inline uint64_t chain_hash_static(
```

Comment:
could the automl_weights_test.cc over in the unit test dir be of any assistance here? Might be easy to cross-reference, but not sure

Reply:
Useful in what sense? This is just the function above but using explicit args and not the things nested in `all`

Reply:
true, just to wire it to a test but seems pretty straightforward.